source/blog/034-request-for-comments-color-spaces.md

```diff
@@ -0,0 +1,261 @@
+---
+title: "Request for Comments: Color Spaces"
+author: Miriam Suzanne and Natalie Weizenbaum
+date: 2022-09-20 19:00 PST
+---
+
+There's been a lot of exciting work in the CSS color specifications lately, and
+as it begins to land in browsers we've been preparing to add support for it in
+Sass as well. The first and largest part of that is adding support for *color
+spaces* to Sass, which represents a huge (but largely backwards-compatible)
+rethinking of the way colors work.
+
+Historically, all colors in CSS have existed in the same color space, known as
+"sRGB". Whether you represent them as a hex code, an `hsl()` function, or a
+color name, they represented the same set of visible colors you could tell a
+screen to display. While this is conceptually simple, there are some major
+downsides:
+
+* As monitors have improved over time, they've become capable of displaying more
+  colors than can be represented in the sRGB color space.
+
+* sRGB, even when you're using it via `hsl()`, doesn't correspond very well with
+  how humans perceive colors. Cyan looks noticeably lighter than purple with the
+  same saturation and lightness values.
+
+* There's no way to represent domain- or device-specific color spaces, such as
+  the [CMYK] color space that's used by printers.
+
+  [CMYK]: https://en.wikipedia.org/wiki/CMYK_color_model
+
+Color spaces solve all of these problems. Now not every color has a red, green,
+and blue channel (which can be interpreted as hue, saturation, and lightness).
+Instead, a every color has a specific *color space* which specifies which
+channels it has. For example, the color `oklch(80% 50% 90deg)` has `oklch` as
+its color space, `80%` lightness, `50%` chroma, and `90deg` hue.
+
+## Color Spaces in Sass
+
+Today we're announcing [a proposal for how to handle color spaces in Sass]. In
+addition to expanding Sass's color values to support color spaces, this proposal
+defines Sassified versions of all the color functions in [CSS Color Level
+4][color-4].
+
+[a proposal for how to handle color spaces in Sass]: https://github.com/sass/sass/blob/main/proposal/color-4-new-spaces.md
+
+### Rules of Thumb
+
+There are several rules of thumb for working with color spaces in Sass:
+
+* The `rgb`, `hsl`, and `hwb` spaces are considered "legacy spaces", and will
+  often get special handling for the sake of backwards compatibility. Colors
+  defined using hex notation or CSS color names are considered part of the
+  `rgb` color space. Legacy colors are emitted in the most compatible format.
```

Comment:
Is it worth noting that CSS is doing roughly the same thing? We're not limiting access to anything with this choice.